### PR TITLE
fix(blind): provided ellipsis for blind header

### PR DESCRIPTION
**Pull request by ridvandmrc**
## 💡 What is the current behavior?
If a long label is used, Ellipsis is not looking .

GitHub Issue Number: #1165

## 🆕 What is the new behavior?

- I removed some css property because it is blocking ellipsis
-

## 🏁 Checklist

A pull request can only be merged if all of these conditions are met (where applicable):

- [X] 🦮 Accessibility (a11y) features were implemented
- [X] 🗺️ Internationalization (i18n) - no hard coded strings
- [X] 📲 Responsiveness - components handle viewport changes and content overflow gracefully
- [X] 📄 Documentation was reviewed/updated (`yarn docs`)
- [X] 🧪 Unit tests were added/updated and pass (`yarn test`)
- [X] 📸 Visual regression tests were added/updated and pass (`yarn visual-regression`)
- [X] 🧐 Static code analysis passes (`yarn lint`)
- [X] 🏗️ Successful compilation (`yarn build`, changes pushed)

### CODE REVIEW

**Comment by danielleroux** [comment]
@ridvandmrc Looks like there are some errors in our visual testing environment. 

You find a html test report on your `Pull Request`-Workflow e.g https://github.com/siemens/ix/actions/runs/8392413573, scroll down to the bottom and check the file `core-html-report--attempt-1 ` to view the report.

To get some information about visual testing checkout our short guide: https://github.com/siemens/ix/blob/main/CONTRIBUTING.md#visual-regression-testing

**Comment by ridvandmrc** [comment]
Hi @danielleroux ,
In order to show the text ellipsis, I had to arrange the width of the content but it is not too much, so I am getting the error. But the changes will not affect anything because the result is:

image

**Comment by danielleroux** [comment]
> Hi @danielleroux , In order to show the text ellipsis, I had to arrange the width of the content but it is not too much, so I am getting the error. But the changes will not affect anything because the result is:
> 
> image

The `sublabel` has to be aligend centered within the blind.
This is also not given by the current implementation:
```tsx
        <ix-blind
          label="Test long long long long long long long 123 456 789"
          sublabel="Sub long long long long long 123 456 789"
          style={{ width: '40rem' }}
        ></ix-blind>
```

image

The label text max with should be half of the blind, following by the sublabel. All labels have to be showing ellipses as text overflow style. If you have access you can checkout the figma specs: https://www.figma.com/file/r2nqdNNXXZtPmWuVjIlM1Q/iX-Components---Brand-Dark?node-id=9598%3A189577

**Comment by danielleroux** [comment]
> Hi @danielleroux , I understand what you said, I updated the code and also It is working as expected.
> 
> image
> 
> Thanks

Thank you.

As mentioned in the PR template, please add some testcases (visual regression tests) to verify the changes.

**Review by danielleroux — CHANGES_REQUESTED** [review]
- Missing visual-regression test cases

**Comment by danielleroux** [comment]
@ridvandmrc Please checkout the [CONTRIBUTING.md](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md#visual-regression-testing). There is a chapter regarding visual tests. You have to run it inside a container.

**Review by danielleroux — APPROVED** [review]

**Review by matthiashader — APPROVED** [review]

File: packages/core/src/components/blind/blind.scss [hunk 1]
```diff
@@ -97,7 +95,6 @@
     position: relative;
     align-items: center;
     flex-grow: 1;
-    flex-shrink: 0;
     margin-right: 1rem;
```
Comment (nuke-ellington) [inline packages/core/src/components/blind/blind.scss:98]:
```suggestion
    margin-right: 0.5rem;
```

**Review by nuke-ellington — CHANGES_REQUESTED** [review]
Updated margin. Please also delete the "blind-sublabel-" snapshots since they are obsolete.

File: packages/core/src/components/blind/blind.scss (hunk 1 above)
Comment (ridvandmrc) [reply packages/core/src/components/blind/blind.scss]: Updated, But I can not update snapshot tests, @danielleroux  Can you please update it for me?

**Review by nuke-ellington — APPROVED** [review]

**Pull request merged**